data/ka'het/ka'het.txt

```diff
@@ -73,6 +73,455 @@ phrase "ka'mar"
+	word
+		". Resource destination: station "
+	word
+		"Seli"
+		"Parii"
+	word
+		", area "
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		"."
+
+phrase "friendly ka'het"
+	word
+		"Primary systems check: "
+	word
+		"shield matrices"
+		"hyperdrive"
+		"life support systems"
+		"energy generator"
+		"weapon systems"
+		"main thrusters"
+		"main steering"
+		"main coolers"
+		"Ravager emitters"
+		"Annihilator loading systems"
+		"tracking systems"
+	word
+		": online. Secondary systems check: "
+	word
+		"solar wind harvester: online and active"
+		"energy storage: available"
+		"support coolers: online and active"
+		"shield restorers: online and active"
+		"long range antennae: available"
+	word
+		"."
+
+phrase "friendly ka'het"
+	word
+		"Current objective: systems "
+	word
+		"patrol"
+		"garrison"
+		"vigilance"
+	word
+		". Threats identified: none. Estimated progress: "
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		"%."
+
+phrase "friendly ka'het"
+	word
+		"Het status: "
+	word
+		"operational"
+		"active"
+		"alive"
+		"functioning"
+		"awake"
+	word
+		". Expected lifetime: "
+	word
+		""
+		"1"
+		"2"
+		"3"
+		"3"
+		"3"
+		"4"
+		"4"
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		" years."
+
+phrase "friendly ka'het"
+	word
+		"Current"
+		"Selected"
+		"Chosen"
+		"Main"
+		"Primary"
+	word
+		" target: <ship>. Government: "
+	word
+		"unknown"
+		"unrecognised"
+		"no data"
+		"unidentified"
+	word
+		". Attitude: "
+	word
+		"friendly"
+		"neutral"
+		"peaceful"
+		"positive"
+		"acceptable"
+		"safe"
+		"tolerable"
+		"adequate"
+	word
+		". No necessary action."
+
+phrase "friendly ka'het"
+	word
+		"Connection with the vessel <ship>: "
+	word
+		"established"
+		"stable"
+		"reliable"
+		"strong"
+	word
+		". Estimated signal strength: "
+	word
+		"8"
+		"9"
+		"9"
+		"9"
+	phrase
+		"digit"
+	word
+		"%."
+
+phrase "friendly ka'het"
+	word
+		"Failed connection with "
+	word
+		"planet Osaeli"
+		"planet Alix"
+		"planet Retilie"
+		"planet Veliante"
+		"planet Eavine"
+		"planet Celeatis"
+		"planet Nona"
+		"planet Palarei"
+
+		# Most of the stations mentioned here don't exist anymore at the time of the player, they were destroyed by the Ka'het during the last 200,000 years
+		"station Ceilia"
+		"station Aerite"
+		"station Letis"
+		"station Maiar"
+		"station Lateria"
+		"station Gertima"
+		"station Santari"
+		"station Antamesa"
+		"construction site Delia"
+	word
+		". Reason: "
+	word
+		"no response"
+		"unknown"
+		"missing response"
+	word
+		". Sending report to station "
+	word
+		"Parii"
+		"Seli"
+	word
+		"."
+
+phrase "friendly ka'het"
+	word
+		"Established connection with station "
+	word
+		"Parii"
+		"Seli"
+	word
+		". Signal strength: "
+	word
+		"3"
+		"4"
+		"5"
+		"6"
+		"7"
+		"8"
+		"9"
+	phrase
+		"digit"
+	word
+		"%. No new data received. Last data received "
+	word
+		""
+		"1"
+		"2"
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		" years, "
+	word
+		""
+		"1 month"
```

Comment:
Would months be an appropriate unit of time for an alien to use? Perhaps if these are said to just be translated, but  having the same 12 months in a year that we do is what's really odd.

Reply:
changed to seasons